### PR TITLE
Include WebPartTitle for SearchBox WebPart

### PR DESCRIPTION
**Pull request by kachihro**
To ensure consistency for the SearchResults, Filter and Verticals - I've included a TITLE for the WebPart for SearchBox.

This is especially useful on a HOME PAGE - when needing to 'send query to new page'.

### CODE REVIEW

**Comment by kachihro** [comment]
Have configured PGP to ensure the code commit is verified

**Comment by kachihro** [comment]
This is the end result - with the ability to have a webpart title - rather than a separate TEXT block (and big space/gap)

image

**Comment by kachihro** [comment]
** How do I sign/verify the 'prior' commits ??

File: search-parts/src/webparts/searchBox/components/SearchBoxContainer.tsx [hunk 1]
```diff
@@ -146,6 +147,16 @@ export default class SearchBoxContainer extends React.Component<ISearchBoxContai
     }
 
     public render(): React.ReactElement<ISearchBoxContainerProps> {
+
+        let renderTitle: JSX.Element = null;
+
+        // WebPart title
+        renderTitle = <WebPartTitle
```
Comment (wobba) [inline search-parts/src/webparts/searchBox/components/SearchBoxContainer.tsx:154]: If a property is not filled out, will this render with no pixels taken?

Comment (kachihro) [reply search-parts/src/webparts/searchBox/components/SearchBoxContainer.tsx:154]: Correct.   I've checked the output/html - and no title has zero impact to the rendered content.   No additional pixels.

image

**Pull request merged**